FabricMC/fabric-docs
# 404 page, localisation improvements

- Implement 404 page
- Add more localisation strings
- Show page authors who contributed outside GitHub
- Update to latest `vitepress-versioning-plugin` version
- Update scrollbar colors
- Move `LICENSE.md` to `LICENSE`
- Improve `contributing.md` file
- Fix miscellaneous Markdown issues
  - Fix weird escapes in some translated links
  - `Codec&amp;amp;lt;Map&amp;amp;gt;` is getting out of hand
- Add `markdownlint-rule-search-replace` rules
  - `missing-heading-anchor`: All headings must have anchors
  - `no-absolute-links`: No absolute links allowed
  - `no-curly-quotes`: No curly quotes allowed
  - `no-file-extension-in-links`: Links should not have an extension
  - `no-multiple-spaces`: Yeah, no multiple spaces,
  - `no-punctuation-in-links`: Punctuation should not be clickable
  - `no-trailing-whitespace`: Extend `MD009` to code blocks
